doc/release-notes/release-notes-4.0.0-rc1.md

```diff
@@ -0,0 +1,252 @@
+Notable changes
```

Comment:
Missing: that this release will activate Canopy. Maybe others? Add it to both this file and the regular release-notes.md so it ends up in the final release.